src/pages/Playground.tsx

```diff
@@ -545,7 +545,7 @@ const Playground: React.FC<ComponentProps> = (props: ComponentProps) => {
               animate={animate}
               setAnimate={setAnimate}
               premium={premium}
-              disableMint={disableMint || blockUI}
```

Comment:
Why remove `blockUI`?

Reply:
I think I used hardcoded value to see the UI working. it should be reverted